### ISSUE TITLE
support expansion with `config()` so user does not have to additionally bring dotenv-expand

### ISSUE THREAD

**Issue by ntodorov** [dotenvx/dotenvx#106]
Instead of using it as cli, I followed the ReadMe to require it in my index.js
```js
require('@dotenvx/dotenvx').config()

console.log(`Hello ${process.env.ENV}`);
console.log(`Hello ${process.env.URL}`);
```
My .env file is
```
ENV=dev
URL=${ENV}
```

The result is
```
Hello dev
Hello ${ENV}
```

From the documentation, it seems that it should have interpolated ${ENV} but it did not.
If I use it as cli it works properly.

**Comment by motdotla** [comment]
released in `v0.22.0`. https://github.com/dotenvx/dotenvx/blob/main/CHANGELOG.md#0220

**Comment by ntodorov** [comment]
Wow that was fast! Thank you!
